### PR TITLE
latest News CLV - remove pin from simple form documentation

### PR DESCRIPTION
**Pull request by samehaidi**

### CODE REVIEW

File: docs/GettingStarted.rst [hunk 1]
```diff
@@ -1290,8 +1286,16 @@ The action is successfully done after confirming it as for the two options above
 
 You may display the home page to verify that the news is available in the appropriate block.
 
+You may display one of the following type of blocks :
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]: Not sure about the term blocks. Here we are talking about different views for the same information. So I would say something like : "You can display the pinned articles with one of the following layouts:". And update "block" by "layout" in the list. WDYT ?

File: docs/GettingStarted.rst [hunk 2]
```diff
@@ -1290,8 +1286,16 @@ The action is successfully done after confirming it as for the two options above
...
+
+- Blog block: 
+
 |image172|
 
+- Latest News block: 
+
+|image198|
+
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst:1298]: Shouldn't we also explain here how to select the layout, and even how to add the portlet with the good parameters (folder, query, ...) ?

**Review by thomasdelhomenie — CHANGES_REQUESTED** [review]

File: docs/GettingStarted.rst (hunk 2 above)
Comment (samehaidi) [reply docs/GettingStarted.rst:1298]: I entend to do it in admin guide as users have no access to this (via administering eXo platform https://docs.exoplatform.org/en/latest/Administration.html#managingpages-editingpage )

Comment (thomasdelhomenie) [reply docs/GettingStarted.rst:1298]: :+1:

File: docs/GettingStarted.rst (hunk 1 above)
Comment (samehaidi) [reply docs/GettingStarted.rst]: Ok. Changes done.

**Review by thomasdelhomenie — APPROVED** [review]

**Pull request merged**